src/WebJobs.Script/Host/AggregateFunctionMetadataProvider.cs

```diff
@@ -155,5 +177,14 @@ internal static FunctionMetadata ValidateBindings(IEnumerable<string> rawBinding
 
             return function;
         }
+
+        private bool IsNullOrEmpty(IEnumerable<RawFunctionMetadata> functions)
```

Comment:
See we already have an extension method in the repo which does this on `IEnumerable<T>` type.